Cargo.lock

```diff
@@ -3514,48 +3838,24 @@ version = "0.42.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "41d2aa71f6f0cbe00ae5167d90ef3cfe66527d6f613ca78ac8024c3ccab9a19e"
 
-[[package]]
-name = "windows_aarch64_msvc"
-version = "0.36.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "9bb8c3fd39ade2d67e9874ac4f3db21f0d710bee00fe7cab16949ec184eeaa47"
```

Comment:
@Vid201 Are you using window for developing. I am not sure if these changes would break anything for you. Please check.

Reply:
I am using Linux. I think we should add Cargo.lock to .gitignore in fact.

Reply:
I am not sure, actually. I think we could keep it.
I just googled with some [reference](https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries).

Reply:
Ok, you're right. Though I saw some projects removing it.

Nevertheless, it doesn't break anything for me, so it's fine.